OwenConferenciasApp/screens/formsLogin/SignInLayout.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {
+  View,
+  TouchableOpacity,
+  TextInput,
+  Image,
+  Text,
+  StyleSheet,
+  Alert,
+} from 'react-native';
+
+export default function SignInLayout() {
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' ,  backgroundColor:'#F0F1F6'}}>
```

Comment:
You can create a type of view block, for example if you want to make a View that you will to use in the app more than one time you can create a style on it, or if you will to use a attibute in a view each time you can put a view that containt other views with the attribute that you will to repeat